dev.opam

```diff
@@ -37,6 +37,9 @@ depends: [
   "caqti-driver-postgresql"
   "caqti-lwt"
   "caqti-async"
+  "async"
```

Comment:
Does not belong to this commit, though yes it might be a good idea to make all direct dependencies explicit.